common/unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -591,7 +591,11 @@ public void writeToOutputStreamIntArray() throws IOException {
     // verify that writes work on objects that are not byte arrays
     final ByteBuffer buffer = StandardCharsets.UTF_8.encode("大千世界");
     buffer.position(0);
-    buffer.order(ByteOrder.LITTLE_ENDIAN);
+    
```

Comment:
Can this just be `buffer.order(ByteOrder.nativeOrder())`? that's what's done elsewhere. 

CC @NathanHowell for a thought.